src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+
+li.forEach((item) => {
+  const span = document.createElement('span');
+
+  span.textContent = item.firstChild.textContent.trim();
+  item.firstChild.textContent = '';
+  item.prepend(span);
+
+  span.addEventListener('click', () => {
+    const childUl = item.querySelector('ul');
+
+    if (childUl) {
+      if (childUl.style.display === 'none') {
+        childUl.style.display = '';
+      } else {
+        childUl.style.display = 'none';
```

Comment:
Instead of checking `childUl.style.display` directly, consider using `window.getComputedStyle(childUl).display` to account for styles applied via CSS. This ensures that the script works correctly even if the initial display state is set via CSS.